### PR TITLE
feat(1307): dev-console - implemented feedback list page

### PR DESCRIPTION
**Pull request by akhilmhdh**
# Closes

1. 1307

# Explain the feature/fix

1. Implemented feedback list page in dev console
2. Fixed a typo made in feedback configuration in console. appId -> app.id

## Does this PR introduce a breaking change

Yes

1. Old feedback configure page is now feedback listing page

## Screenshots

<details>
<summary>View Screenshots</summary>

## Feedback Listing

Screenshot 2022-01-21 at 3 52 20 PM

## Feedback Loading

Screenshot 2022-01-21 at 3 52 30 PM

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/developer-console-spa/src/@types/Feedback/types.d.ts [hunk 1]
```diff
@@ -9,6 +9,45 @@ declare type FeedbackConfig = {
...
+  OPEN = "OPEN",
+}
+
+type FeedbackUserProfile = {
+  cn: String;
+  name: string;
+  uid: string;
+  rhatUUID: string;
+  email: string;
+  url: string;
```
Comment (riginoommen) [inline packages/developer-console-spa/src/@types/Feedback/types.d.ts:28]: Seems to be unused types

File: packages/developer-console-spa/src/@types/Feedback/types.d.ts [hunk 2]
```diff
@@ -9,6 +9,45 @@ declare type FeedbackConfig = {
...
+  source: string;
+  ticketUrl: string;
+  category: FeedbackCategory;
+  module: string;
+  stackInfo: StackType;
+  assignee: FeedbackUserProfile;
+  createdOn: string;
+  createdBy: string | FeedbackUserProfile;
+  updatedOn: string;
+  updatedBy: string | FeedbackUserProfile;
```
Comment (riginoommen) [inline packages/developer-console-spa/src/@types/Feedback/types.d.ts:48]: Conflict with the timestamp time for createdOn and updatedOn

File: packages/developer-console-spa/src/@types/Feedback/types.d.ts (hunk 1 above)
Comment (akhilmhdh) [reply packages/developer-console-spa/src/@types/Feedback/types.d.ts:28]: It's used just below

File: packages/developer-console-spa/src/@types/Feedback/types.d.ts (hunk 2 above)
Comment (akhilmhdh) [reply packages/developer-console-spa/src/@types/Feedback/types.d.ts:48]: Sorry, didn't get you

**Review by deshmukhmayur — APPROVED** [review]

**Review by riginoommen — APPROVED** [review]

**Pull request merged**